test/onnx/test_fx_op_consistency.py

```diff
@@ -162,6 +162,12 @@
     ]
 )
 
+# NOTE: For ATen signature modifications that will break ONNX export,
+# use **xfail_onnxscript** and **skip_onnxscript** instead of xfail or skip
+# to make the signal apparent for maintainers.
+xfail_onnxscript = xfail
```

Comment:
nit: name bikeshedding, should we callout "bc break", "aten modification" in the name? The current name alone feels like due to general limitation on onnxscript, which is kinda accurate too, but overshadows the original context.

Reply:
xfail_torchlib_bc_break ?

Reply:
I think this _may_ be too restrictive. There are other cases when tests will break. For example, if the op emits a different aten we don't account for, it will also break too but it is hard to put it in the aten modification category. For bc break I think it is also not entirely accurate, because (1) the modifed aten doesn't break bc. (2) torchlib doesn't break bc either. It is more torchlib doesn't have forward compatibility.

Maybe xfail_torchlib_forward_compatility if we want to be explicit?

Reply:
Updated